CMakeLists.txt

```diff
@@ -110,4 +111,6 @@ rosidl_generate_interfaces(${PROJECT_NAME}
     trajectory_msgs
 )
 
+ament_export_dependencies(rosidl_default_runtime)
```

Comment:
Adding `<exec_depend>rosidl_default_runtime</exec_depend>` to `package.xml` is in the migration guide, but I see nothing about this. Could you please clarify why this is needed?

Reply:
This is taken from the ros2 [example_interfaces](https://github.com/ros2/example_interfaces) repo. It is used in nearly every srv and msg repo as best as I can tell. That said, the package itself only contains a [cmake file](https://github.com/ros2/rosidl_defaults/blob/master/rosidl_default_runtime/rosidl_default_runtime-extras.cmake.in) and I'm not entirely sure that it is needed